morphocut/stream.py

```diff
@@ -17,8 +17,9 @@
 @ReturnOutputs
 class TQDM(Node):
     """
-    Provide a progress indicator via `tqdm`_. It prints a dynamically
-    updating progressbar.
+    Provide a progress indicator via `tqdm`_.
+    
+    Show a dynamically updating progressbar using `tqdm`_.
```

Comment:
Only use my suggestion (the latter) here. Both sentences say the same but mine is more complete. Actually, it's "progress bar", my bad.